rs/execution_environment/benches/wasm_instructions/helper.rs

```diff
@@ -43,21 +88,41 @@ pub fn benchmark_with_loop_confirmation(name: &str, code: &str) -> Vec<common::B
     let c = CONFIRMATION_LOOP_ITERATIONS;
     let r = DEFAULT_REPEAT_TIMES;
     vec![
-        benchmark(name, i, r, code),
-        benchmark(&format!("{name}/confirmation"), c, r, code),
+        benchmark(&format!("wasm32/{name}"), i, r, code, Wasm64::Disabled),
+        benchmark(
+            &format!("wasm32/{name}/confirmation"),
+            c,
+            r,
+            code,
+            Wasm64::Disabled,
+        ),
+        benchmark(&format!("wasm64/{name}"), i, r, code, Wasm64::Enabled),
+        benchmark(
+            &format!("wasm64/{name}/confirmation"),
+            c,
+            r,
+            code,
+            Wasm64::Enabled,
+        ),
     ]
 }
 
 /// Creates a benchmark with a code block repeated specified number of times in a loop.
-pub fn benchmark(name: &str, i: usize, r: usize, repeat_code: &str) -> common::Benchmark {
+pub fn benchmark(
+    name: &str,
+    i: usize,
+    r: usize,
+    repeat_code: &str,
+    wasm64_enabled: Wasm64,
+) -> common::Benchmark {
```

Comment:
We could replace the `memtype` here for all the benchmarks, including the `benchmark_with_loop_confirmation`:

```suggestion
) -> common::Benchmark {
    let wasm64_enabled = name.starts_with("wasm64");
    let code = if wasm64_enabled {
        code.replace("memtype", "i64")
    } else {
        code.replace("memtype", "i32")
    };
```

Reply:
okay, thanks! solved now.